src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -73,9 +75,59 @@ type CA struct {
 	Cert       string // PEM-encoded
 }
 
+func exists(file string) bool {
+	if _, err := os.Stat(file); os.IsNotExist(err) {
+		fmt.Printf("no such file or directory: %s", file)
+		return false
+	}
+	return true
+}
+
+func configExists(file string) (string, bool) {
+	configPath, err := InConfigDir(file)
+	if err != nil {
+		log.Errorf("Could not get config path? %v", err)
+		return configPath, false
+	}
+	return configPath, exists(configPath)
+}
+
+func majorVersion(version string) string {
+	return r.FindString(version)
+}
+
+// copyNewest is a one-time function for using older config files in the 2.x series.
+// from 2.0.2 forward, Lantern will consider all major versions to be compatible and
+// will name them accordingly, as in "lantern-2.yaml".
+func copyNewest(file string) {
+	// If we already have a config file with the latest name, use that one.
+	// Otherwise, copy the most recent config file available.
+	if cur, exists := configExists(file); exists {
+		return
+	} else if path, exists := configExists("lantern-2.0.1.yaml"); exists {
+		if err := os.Rename(path, cur); err != nil {
+			log.Errorf("Could not rename file: %v", err)
+		}
+	} else if path, exists := configExists("lantern-2.0.0+stable.yaml"); exists {
+		if err := os.Rename(path, cur); err == nil {
```

Comment:
Could loop through an array of file names to make code more clear, but nothing important.